### PR TITLE
Took the due date to right side for better design and also added over…

### PR DESCRIPTION
**Pull request by devalcodes**
Hi @Yuvraj0157,

I suggest moving the due date to the right, as shown in the picture, to accommodate longer to-do items that users may input.
I've added an "overflow: hidden" to ensure the UI remains clean and visually appealing, even with lengthy to-do items. Users can access the full text via a tooltip when hovering over the task.
I believe these changes will improve both the functionality and the overall cleanliness of the UI.

Best regards,
Devalnath
Screenshot from 2024-10-07 14-40-55
Screenshot from 2024-10-07 14-41-31
Screenshot from 2024-10-07 14-45-58

### CODE REVIEW

**Comment by devalcodes** [comment]
I was trying to resolve #1

File: middleware/tokenMiddleware.js
Comment (Yuvraj0157) [inline middleware/tokenMiddleware.js:1]: this change is unnecessary.

File: routes/auth.js
Comment (Yuvraj0157) [inline routes/auth.js:1]: no change is required here.

File: server.js
Comment (Yuvraj0157) [inline server.js:1]: no change is required here

**Comment by devalcodes** [comment]
Ya I will resolve this and update, is the rest fine?

**Comment by devalcodes** [comment]
All unnecessary changes have been resolve

**Comment by devalcodes** [comment]
All unnecessary changes have been resolved, do review and let me know

**Comment by devalcodes** [comment]
Hello @Yuvraj0157 Any Updates?

**Comment by Yuvraj0157** [comment]
Looks good, the only issue is the toggle checkbox. Restore its behavior(change the size as it was earlier, hover property)

**Comment by devalcodes** [comment]
It seemed really small that's why I changed it, no issues I'll revert it

**Comment by Yuvraj0157** [comment]
The earlier checkbox size was bigger. 
image

In proposed PR:
image

**Comment by devalcodes** [comment]
Yeah, I'll revert it to original size no issues

**Comment by devalcodes** [comment]
[Screencast from 2024-10-09 15-41-06.webm](https://github.com/user-attachments/assets/c5c9d09e-8ba1-4d69-b1c3-06385f5abedb)

Checkbox has been made bigger, if everything is alright do kindly merge and close it

**Pull request merged**